src/core/services/layout/layout.js

```diff
@@ -367,6 +375,30 @@
   }
 
   /**
+   * Centralize warnings for known flexbox issues (especially IE-related issues)
+   */
+  function validateAttributeUsage(className, attr, element, $log){
```

Comment:
I would lean towards not actually doing this with a warning, and instead just having it in the documentation. This adds payload size and runtime cost for something that isn't necessary for the feature to function.